apisix/plugins/error-log-logger.lua

```diff
@@ -36,23 +38,40 @@ local lrucache = core.lrucache.new({
 local metadata_schema = {
     type = "object",
     properties = {
-        host = schema_def.host_def,
-        port = {type = "integer", minimum = 0},
-        tls = {type = "boolean", default = false},
-        tls_server_name = {type = "string"},
-        timeout = {type = "integer", minimum = 1, default = 3},
-        keepalive = {type = "integer", minimum = 1, default = 30},
+        type = {type = "string", default = "TCP", enum = {"TCP", "SKYWALKING"}},
```

Comment:
This is a definitely broken change, we may need some further discussions.

Reply:
I will revert the remove operation and mark them as depreceated.